test/common/tests.go

```diff
@@ -9,21 +9,21 @@ var (
 
 	AFTER_INSTALL_TESTS = []TestCase{
 		// Test to be execute after RHMI has been installed
```

Comment:
Could you put a comment here with a link to the test cases please

Reply:
@davidffrench I've added a comment on the file header